sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/__init__.py

```diff
@@ -13,12 +13,14 @@
 
 try:
     from ._patch import __all__ as _patch_all
-    from ._patch import *  # type: ignore # pylint: disable=unused-wildcard-import
+    from ._patch import *  # pylint: disable=unused-wildcard-import
 except ImportError:
     _patch_all = []
 from ._patch import patch_sdk as _patch_sdk
 
-__all__ = ["ConfidentialLedgerClient"]
+__all__ = [
+    "ConfidentialLedgerClient",
```

Comment:
I actually did it on purpose because I thought we could take the occasion to update the auto-generated code. But you're right, it makes sense the PR too messy to review as there are much more changes than I realized. I will remove these changes then.